docker/Dockerfile

```
@@ -1,3 +1,31 @@
+WORKDIR /build
+
+# Install build dependencies
+RUN apk add --no-cache \
+    build-base \
+    pkgconfig \
+    openssl-dev \
+    openssl-libs-static \
+    git
+
+# Clone m4b-merge from source
+RUN git clone --depth 1 --branch develop https://github.com/djdembeck/m4b-merge.git /build/m4b-merge
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)